### DISCUSSION TITLE
Workflow condition "Contains"

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15804] [General] [answered]
As per image below, workflows starting conditions does not have "Contains" trigger which would work with multiselect picklist. 
I think vtiger workflows supports it. Here it seems to be very useful. Any workaround? 

![image](https://user-images.githubusercontent.com/84110093/159414897-0fa06d09-f988-4476-96fa-71eb87f9844b.png)

**Comment by davide-alghi** [comment, marked as answer]
Hi @s2b-git,
the "contains" operator is a String operator.
Picklist (single or multiple) is an Enum value.
yes, vTiger provides "contains" as a picklist operator.
Solution: You need to create multiple conditions in "Any Condition" group each with a picklist value that needs to be matched.
Alternatively you can create multiple conditions in "All Condition" group each with a picklist value that doesn't need to be matched.
